USER_GUIDE.md

```diff
@@ -1,40 +1,65 @@
 ## User Guide
 
+Hi, 😄 welcome to the User's guide to using react-native-use-styles. With `react-native-use-styles` you can easily de-clutter 💆‍♂️ your style code and write a better and more efficient styling code 🤩.
```

Comment:
Sorry I didn't saw this but we should use useStyles instead of `react-native-use-styles`:

```suggestion
Hi, 😄 welcome to the User's guide to using useStyles. With useStyles you can easily de-clutter 💆‍♂️ your style code and write a better and more efficient styling code 🤩.
```